### DISCUSSION TITLE
How important is stability on bracketed exposures?

### DISCUSSION THREAD

**Discussion by Maxli-Cat** [EatTheFuture/image_tools#9] [Questions & Troubleshooting] [answered]
Is it important that bracketed exposures are tripod stable or is handheld pointed at the same scene still usable?

**Comment by cessen** [comment, marked as answer]
### Short answer:
You definitely want them to be as locked down as you can.  Ideally that means using a tripod, but you could probably get away with placing the camera on a table or something and being careful not to jostle it between shots.

### Long answer:

The algorithm used is pretty robust, but it does assume that the over-all colors don't change between shots.

Basically, the more different your shots are between exposures, the worse the estimate will be.  Subtle changes like rustling of grass or ripples in water are unlikely to cause major issues.  But letting the framing of the shot itself change actually causes much larger changes to the over-all color distribution of the image than you might think.

Imagine, for example, an image that has both land and sky in it: even just a 10% vertical shift down means that 10% of the sky is disappearing and getting replaced by 10% more land, causing a significant over-all darkening of the image.  And that can throw off the estimate really badly.
